### DISCUSSION TITLE
2023-11-29T08:22:36Z - The remote server returned an error: (500) Internal Server Error.

### DISCUSSION THREAD

**Discussion by almadorinc** [Sellbrite/Sellbrite-API#28] [Q&A]
Hi,

We had a developer create a program to upload tracking to sellbrite using API. This has been working for years until now - we get this error:

2023-11-29T08:22:36Z - The remote server returned an error: (500) Internal Server Error.

We haven't changed anything. Are there any changes on sellbrite's end that we should be aware of?

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

I don’t think there have been any significant changes. Are you able to post the full request with the body params? That might help us see what’s causing the internal server error

**Comment by almadorinc** [comment]
Hi,

We don't have the params because it is inside an exe file, but here is some
code from another script that is also giving us a 500 error:

Dim http As New XMLHTTP60
Dim endpoint As String
Dim apiKey As String
Dim apiSecret As String
Dim auth As String
Dim rateData As String
account_token = "****"
secret_key = "*****"
auth = account_token & ":" & secret_key
auth = "Basic " & encodeBase64(account_token & ":" & secret_key)
'**********   Range("E1").value
'''"{
'''    ""shipment"": {
'''        ""sb_order_seq"": 1225675,
'''        ""carrier_name"": ""USPS"",
'''        ""tracking_number"": ""9434611206210829208389"",
'''        ""items"": [
'''            1,
'''            {
'''                ""sku"": ""4983164378580"",
'''                ""quantity"": 1
'''            }
'''        ],
'''        ""shipping_method"": ""USPSGroundAdvantage™"",
'''        ""warehouse_uuid"": ""b0f4d424-de57-4a8a-af1a-0e172acc8ed4"",
'''        "" tracking_url"": ""
https://tools.usps.com/go/TrackConfirmAction?qtc_tLabels1=9434611206210829208389
""
'''    }
'''}"
'**********   Range("E1").value
rateData = Range("E1")
endpoint = "https://api.sellbrite.com/v1/shipments" ' & queryParams
http.Open "POST", endpoint, False
http.setRequestHeader "Authorization:", auth
http.setRequestHeader "Content-Type", "application/json"
http.send rateData
  While http.ReadyState <> 4
        DoEvents
    Wend
If http.Status = 200 Then
        Debug.Print http.responseText
    Else

        Debug.Print "Error: " & http.Status & " - " & http.StatusText
    End If
